tests/api.v2/middlewares/authMiddlewares.test.js

```diff
@@ -92,15 +99,41 @@ describe('Tests for authorization/authentication middlewares', () => {
     expect(next).toBeCalledWith(expect.any(UnauthenticatedError));
   });
 
+  it('Express middleware can reject users that didnt agree to privacy policy', async () => {
+    UserAccessModel.canAccessExperiment.mockImplementationOnce(() => true);
+
+    const req = {
+      params: { experimentId: fake.EXPERIMENT_ID },
+      url: fake.RESOURCE_V1,
+      user: {
+        sub: 'allowed-user-id',
+      },
+      method: 'POST',
+    };
+    const next = jest.fn();
+
+    await expressAuthorizationMiddleware(req, {}, next);
+    expect(next).toBeCalledWith(expect.any(NotAgreedToTermsError));
+  });
+
   it('expressAuthenticationOnlyMiddleware works correctly', async () => {
     const next = jest.fn();
-    const req = { user: 'someuserid-xd-123' };
+    const req = { user: { sub: 'someuserid-xd-123' } };
     await expressAuthenticationOnlyMiddleware(req, {}, next);
     expect(next).toBeCalled();
   });
+
   it('expressAuthenticationOnlyMiddleware should fail if req.user is empty', async () => {
     const next = jest.fn();
     const req = { user: null };
     await expect(expressAuthenticationOnlyMiddleware(req, {}, next)).rejects;
   });
+
+  it('expressAuthenticationOnlyMiddleware should fail if privacy policy wasnt agreed on', async () => {
```

Comment:
I don't add the `` '  `` in the tests titles because the linter auto changes them to `' \' '`. When that happens and e.g. I get informed that "policy wasn't agreed on" failed I can't just copy paste and search for it directly because I need to first add the escape characters to the title. If I don't add the `'` all this problem disappears